packages/cashier/src/stores/__tests__/account-transfer-store.spec.tsx

```diff
@@ -1,47 +1,88 @@
 import AccountTransferStore from '../account-transfer-store';
 import { getCurrencies, validNumber, CFD_PLATFORMS } from '@deriv/shared';
+import { DeepPartial, TRootStore, TWebSocket, TTransferAccount, TMT5LoginAccount } from '../../types';
 
 let accounts, account_transfer_store, root_store, WS;
 
-const CR_eUSDT_account = {
+const CR_eUSDT_account: TTransferAccount = {
     account_type: 'trading',
     balance: '1.00000000',
     currency: 'eUSDT',
     demo_account: 0,
     loginid: 'CR90000113',
 };
 
-const CR_USD_account = {
+const CR_USD_account: TTransferAccount = {
     account_type: 'trading',
     balance: '10.00',
     currency: 'USD',
     demo_account: 0,
     loginid: 'CR90000103',
 };
-const MT_USD_account = {
+const MT_USD_account: TTransferAccount = {
     account_type: 'mt5',
     balance: '10.00',
     currency: 'USD',
     demo_account: 0,
     loginid: 'MT0000000',
 };
 
-const MX_USD_account = {
+const MX_USD_account: TTransferAccount = {
     account_type: 'trading',
     balance: '10.00',
     currency: 'USD',
     demo_account: 0,
     loginid: 'MX0000000',
 };
 
-const DXR_USD_account = {
+const DXR_USD_account: TTransferAccount = {
     account_type: 'dxtrade',
     balance: '10.00',
     currency: 'USD',
     loginid: 'DXR1008',
     market_type: 'financial',
 };
 
+const trading_platform_accounts: Array<TMT5LoginAccount> = [
+    {
+        account_id: 'DXR1002',
+        account_type: 'real',
+        balance: 0,
+        currency: 'USD',
+        login: '52',
+        market_type: 'synthetic',
+    },
+    {
+        account_id: 'DXR1003',
+        account_type: 'real',
+        balance: 0,
+        currency: 'USD',
+        login: '52',
+        market_type: 'financial',
+    },
+];
+
+const mt5_login_list: Array<TTransferAccount> = [
+    {
+        ...MT_USD_account,
+        loginid: 'MTR111176',
+        market_type: 'financial',
+        sub_account_type: 'financial',
+    },
+    {
+        ...MT_USD_account,
+        loginid: 'MTR111177',
+        market_type: 'financial',
+        sub_account_type: 'financial',
+    },
+    {
+        ...MT_USD_account,
+        loginid: 'MTR40000265',
+        market_type: 'synthetic',
+        sub_account_type: 'financial',
+    },
+];
+
 beforeEach(() => {
     accounts = [
```

Comment:
just a nitpick: add please also types for account and account_transfer_store :)

Reply:
Done